api/src/main/java/ru/urururu/util/Coverage.java

```diff
@@ -0,0 +1,52 @@
+package ru.urururu.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * @author <a href="mailto:dmitriy.g.matveev@gmail.com">Dmitry Matveev</a>
+ */
+public class Coverage {
+    private static final Map<File, List<Integer>> coverage = new HashMap<>();
+
+    public synchronized static void markAsCode(File file, int line) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)